jenkins/bootstrap.py

```diff
@@ -1080,5 +1086,6 @@ def parse_args(arguments=None):
 
 
 if __name__ == '__main__':
+    logging.info('Args: %s', ' '.join(pipes.quote(a) for a in sys.argv[1:]))
```

Comment:
I think this messed up logging, since it is called before `setup_logging()`. only lines of level WARNING or higher are being printed now, and the line prefixes are wrong, too.